d2bs/kolbot/libs/common/Misc.js

```diff
@@ -69,7 +68,6 @@ var Skill = {
 		case 101: // Holy Bolt
 		case 107: // Charge
 		case 130: // Howl
-		case 132: // Leap
```

Comment:
same here, 132/leap was specified twice, but has more accurate calculations in the other case